gamemodes/terrortown/entities/weapons/weapon_tttbasegrenade.lua

```diff
@@ -192,8 +334,8 @@ function SWEP:Throw()
 		tang.p = math.Clamp(tang.p,-90,90) -- Makes the grenade not go backwards :/
 
 		local vel = math.min(800, (90 - tang.p) * 6)
-		local thr = tang:Forward() * vel + ply:GetVelocity()
-		self:CreateGrenade(src, Angle(0,0,0), thr, Vector(600, math.random(-1200, 1200), 0), ply)
+		local force = tang:Forward() * vel + ply:GetVelocity()
+		self:CreateGrenade(src, Angle(0,0,0), force, Vector(600, math.random(-1200, 1200), 0), ply)
```

Comment:
I think you want to replace the code above with the `GetThrowVelocity` function

Reply:
fixed, good spotting